src/sentry/migrations/0280_backfill_dashboard_widget_limit.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.24 on 2022-03-10 14:52
+
+from django.db import migrations
+
+from sentry.models import DashboardWidgetDisplayTypes
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+def backfill_dashboard_widget_limit(apps, schema_editor):
+    DashboardWidget = apps.get_model("sentry", "DashboardWidget")
+
+    for widget in DashboardWidget.objects.all():
```

Comment:
In general you should use `RangeQuerySetWrapperWithProgressBar` here to avoid bringing rows into memory at once.